frontend/models/product-list/types.ts

```diff
@@ -36,6 +36,7 @@ export enum ProductListType {
    AllTools = 'tools',
    ToolsCategory = 'tools-category',
    Marketing = 'marketing',
+   Store = 'store',
```

Comment:
I doesn't "feel" right to add "store" as a product list type, since store is not a product list page. Can we refactor the `ProductListBreadcrumb` component to account for non-product list segments?

There's a more generic `PageBreadcrumb` component that has been used in the new product page and could be used here too, but maybe this would be a bigger refactor than it needs to be for the scope of this PR. 🤔